js/localization/messages/en.json

```diff
@@ -327,10 +327,10 @@
         "dxHtmlEditor-alignRight": "Align Right",
         "dxHtmlEditor-alignJustify": "Align Justify",
         "dxHtmlEditor-codeBlock": "Code Block",
-        "dxHtmlEditor-variable": "Variable",
+        "dxHtmlEditor-variable": "Add Variable",
         "dxHtmlEditor-undo": "Undo",
         "dxHtmlEditor-redo": "Redo",
-        "dxHtmlEditor-clear": "Clear",
+        "dxHtmlEditor-clear": "Clear Formats",
```

Comment:
In Word this control has the following name:

```suggestion
        "dxHtmlEditor-clear": "Clear Formatting",
```

Is this the same thing?

Reply:
Yep. By the other hand excel has the `Clear Formats`:
![image](https://user-images.githubusercontent.com/1554153/117049599-ff5e6d00-ad1c-11eb-8dc2-f21837b9e7d1.png)